Sources/ArcGISToolkit/Documentation.docc/FloatingPanel.md

```diff
@@ -18,7 +18,8 @@ The following images are of a simple list of numbers in a floating panel.
 | ------ | ---- |
 | ![image](https://user-images.githubusercontent.com/3998072/202795901-b86d6d26-3572-4c88-8f6e-84473ce57002.png) | ![image](https://user-images.githubusercontent.com/3998072/202796009-92e3b5c3-d88b-4124-8d9f-bad6df445f02.png) |
 
-- Note: The Floating Panel is exposed as a view modifier. See ``SwiftUI/View/floatingPanel(attributionBarHeight:backgroundColor:selectedDetent:horizontalAlignment:isPresented:maxWidth:_:)``
+- Note: The Floating Panel is exposed as a view modifier. See 
+``SwiftUICore/View/floatingPanel(attributionBarHeight:backgroundColor:selectedDetent:horizontalAlignment:isPresented:maxWidth:_:)`` or ``SwiftUICore/View/floatingPanel(attributionBarHeight:selectedDetent:horizontalAlignment:isPresented:maxWidth:_:)`` instead.
```

Comment:
```suggestion
- Note: The Floating Panel is exposed as a view modifier. See 
``SwiftUICore/View/floatingPanel(attributionBarHeight:backgroundColor:selectedDetent:horizontalAlignment:isPresented:maxWidth:_:)``.
```
My previous suggestion introduced a DocC warning. Should only one instance method be documented here because `backgroundColor` has a default value?

Reply:
Looks like this is dependent on whether you're building the doc for iOS/Catalyst or visionOS. What's more is that it doesn't seem we can use compiler directives here. I'll look for a workaround.

Reply:
See [ba38ac5](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/964/commits/ba38ac592cf96f5054348958f47c67325660127e). It's not as good as a direct link but best workaround I've found so far.